src/content/app/genome-browser/state/drawer/drawer-sequence/drawerSequenceSelectors.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { RootState } from 'src/store';
+
+export const isDrawerSequenceVisible = (state: RootState, genomeId: string) => {
+  return state.browser.drawer.sequence[genomeId]?.isVisible ?? false;
```

Comment:
Why is nullish coalescing needed here when `isVisible` is of type `boolean`?

Reply:
Because if `state.browser.drawer.sequence[genomeId]` doesn't exist, `state.browser.drawer.sequence[genomeId]?.isVisible` will be undefined.